elements/lisk-p2p/src/peer_directory/peer_list.ts

```diff
@@ -28,8 +28,7 @@ export interface CustomPeerInfo {
 
 export interface AddPeerOutcome {
 	readonly success: boolean;
-	readonly isEvicted: boolean;
-	readonly evictedPeer?: P2PPeerInfo;
+	readonly evictedPeer: P2PPeerInfo | undefined;
```

Comment:
I prefer `isEvicted`. The new alternative is redundant -> `AddPeerOutcome.evictedPeer`.